### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ThunderDeliverer**
# Description

As the README is a developer's first interaction with our smart contracts, it needed to be updated to better represent the current state of our development.

# Actions

The repository README has been updated to reflect the current state of the project.

# Checklist

- [N/A] Verified code additions
- [N/A] Updated NatSpec comments (if applicable)
- [N/A] Regenerated docs
- [N/A] Ran prettier
- [N/A] Added tests to fully cover the changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,119 @@
...
 
-**NOTE: Please be aware that RMRKEquippable is likely very close to the maximum contract deployment size allowed by most
-EVM environments. If you need more space, consider RMRKNestingExternalEquippable.**
+### Nestable ([RMRKNestable](./contracts/RMRK/nestable/RMRKNestable.sol)): [EIP-6059: Parent-Governed Nestable Non-Fungible Tokens](https://eips.ethereum.org/EIPS/eip-6059)
 
-## Emotable
+1. Deploy like a regular ERC-721 compliant smart contract.
 
-> TBD
+That's it! This contract can recieve and be nested by other instances of RMRKNestable.
```
Comment (steven2308) [inline README.md:63]:
```suggestion
That's it! This contract can receive and be nested by other instances of RMRKNestable.
```

File: README.md [hunk 2]
```diff
@@ -1,63 +1,119 @@
...
+## Interfaces
+
+The interfaces of all of the smart contracts are included in this repository. They are prefixed with an `I`; so the
+interface for the `RMRKNestable` is `IRMRKNestable`. If you wish to interact with any of the RMRK smart contracts from
+your own, you only need to import the desired interface.
+
+## Implementations
+
+In addition to the raw RMRK legos and extensions, this repository also contains implementations of all of them in the
+[`implementations/`](./contracts/implementations/) directory. These implementations are oppinionated and utilize the
```
Comment (steven2308) [inline README.md:113]:
```suggestion
[`implementations/`](./contracts/implementations/) directory. These implementations are opinionated and utilize the
```

File: README.md [hunk 3]
```diff
@@ -81,55 +137,4 @@ Logic should go into a Logic field of the NFT's body, and is executed exclusivel
...
-Then, copy the deployment address and paste it in to replace `DEPLOYED_CONTRACT_ADDRESS` in this command:
-
-```shell
-npx hardhat verify --network goerli DEPLOYED_CONTRACT_ADDRESS "Hello, Hardhat!"
-```
-
-# Performance optimizations
-
-For faster runs of your tests and scripts, consider skipping ts-node's type checking by setting the environment variable `TS_NODE_TRANSPILE_ONLY` to `1` in hardhat's environment. For more details see [the documentation](https://hardhat.org/guides/typescript.html#performance-optimizations).
+An extension for the contracts to make them Harberger-taxable by default, integrating the selling and taxing functionality right into the NFT's mint. This does mean the NFT can never not be Harb-taxed, but there can be an on-off flag for this that the _ultimate owner_ (a new owner type?) can flip. -->
```
Comment (steven2308) [inline README.md:140]: Dafuq is this? :laughing:

**Review by steven2308 — CHANGES_REQUESTED** [review]
Only typos and some extra piece that slipped there.
Looks great!

**Comment by steven2308** [comment]
On more thing, I think on the equippable section you go ahead and explain how to deploy the split version only, and not how to do the merged one. I think it's good to have both.

File: README.md (hunk 3 above)
Comment (ThunderDeliverer) [reply README.md:140]: I think this was a remainder of initialising an empty Hardhat project 😅

**Review by steven2308 — APPROVED** [review]
Let's gooo

**Pull request merged**